types/constants.go

```diff
@@ -39,6 +45,11 @@ var (
 	// The GenesisID is used in many places. Calculating it once saves lots of
 	// redundant computation.
 	GenesisID BlockID
+
+	// Oak hardfork constants.
+	OakHardforkBlock     BlockHeight
+	OakDifficultyMaxRise = big.NewRat(1004, 1000)
+	OakDifficultyMaxDrop = big.NewRat(1000, 1004)
```

Comment:
The existing difficulty adjustment values live in `types` already, using the names `MaxAdjustmentUp` and `MaxAdjustmentDown`.

Things belong in the `types` package if other, non-consensus packages might need to see the constants, because they can't necessarily import the consensus package (in practice they can).

I'll think about package placement a bit more, but I think the  types package is the right place for the hardfork height.